src/bindings/python/requirements.txt

```diff
@@ -1 +1 @@
-numpy>=1.16.6,<=1.23.1
+numpy>=1.16.6
```

Comment:
```suggestion
numpy>=1.16.6,<=1.23.4
```
Please do it in all following `numpy` cases. Similar for Tensorflow.

We had discussed it together today.

Reply:
Let us discuss offline the next steps. I would not create super story from this task.
Let us negotiate with QA team about the next steps

Reply:
FIxed.